game/analyzer.hh

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "ringbuffer.hh"
+#include "tone.hh"
+
+#include <cstdint>
+#include <complex>
+#include <vector>
+#include <list>
+#include <algorithm>
+#include <cmath>
+
+static const unsigned FFT_P = 10;
+static const std::size_t FFT_N = 1 << FFT_P;
+
+/// analyzer class
+ /** class to analyze input audio and transform it into useable data
+ */
```

Comment:
Weird comment structure (yes it was already like this but since you've touched it now.... :P)
1. Remove the superfluous comment `///analyzer class`
2. make the description of the class single line since it fits on one line

Reply:
Good point. Thanks